cluster/examples/kubernetes/ceph/cluster-test.yaml

```diff
@@ -17,6 +17,7 @@ data:
     [global]
     osd_pool_default_size = 1
     mon_warn_on_pool_no_redundancy = false
+    bdev_flock_retry = 20
```

Comment:
If this stabilizes the tests, any reason not to make it a default in the product as well?

Reply:
The default is 3 in Ceph which has proven to be sufficient so far, at least outside of this CI resource constraint env.